src/input.c

```diff
@@ -107,6 +108,8 @@ VbResult input_keypress(Client *c, int key)
 
 VbResult input_open_editor(Client *c)
 {
+    static unsigned long element_map_next_key = 1;
+    unsigned long element_map_key = 0;
```

Comment:
I do not see the benefit in having two variables to increment a counter. Wouln't it be enought to use only `static usigned long` and increment it?

Reply:
The idea behind the second variable is for a clean code flow.

The default value of `0` is set during initialization.
The only logic concerning its change is within the code block that handles the special case `element does not have an id`

althought to be true to this I should have applied the same logic to the `c->element_id` assignment

Reply:
It's your choice. I didn't spent much tinking about, but I think we could use a JavaScript function or two to do the dirty stuff for us. So that we only need to call these functions with the `element_map_key` as param.
It' not necessary to put more and more changes into this pull request. If it works, I'll merge it. We can than cleanup some things afterwards.

Reply:
I agree. A vimb javascript object would be nice to keep the javascript organized and injected at a generalized place. On the other hand those javascript functions are also open to be called by the pages' javascript so that might be a weird security concern. Probably too small of a user base to be of any interest thought.
Anyway for now I have pushed the latest version of my code. I've been actively using it for the last few days along with my tabbed patch and I have not had any problems so far.